en/setup/dev_env_linux_arch.md

```diff
@@ -1,27 +1,48 @@
-# Development Environment on ArchLinux
+# Development Environment on Arch Linux
 
-> **Note** These instructions allow you to build PX4 (without RTPS) for NuttX targets, using an unsupported version of GCCE from the package manager.
-The instructions have been tested on Antergos (an Arch Linux based distribution) as it is easier to set up than Arch Linux.
-We hope to provide fully tested instructions with the supported toolchain in the near future.
+> **Note** These instructions allow you to build PX4 (without RTPS) for NuttX targets and run simulation with jMAVsim or gazebo.
+The instructions have been tested on Manjaro (an Arch Linux based distribution) as it is much easier to set up than Arch Linux.
 
-## Permissions
+## Setup script
 
-The user needs to be added to the group "uucp":
+Using the the provided setup script it's very convenient to set your Arch installation up for PX4 development. Here are the steps to get you started from scratch.
 
+### Run the setup script
+The script by default installs all tools to build PX4 (without RTPS) for NuttX targets and run simulation with jMAVsim.
+
+Optionally you can specify the parameter `--gazebo` to also install gazebo simulation from the [AUR](https://aur.archlinux.org/packages/gazebo/). Note that gazebo gets compiled from source and therefore takes some time to install and requires entering the sudo password multiple times for dependencies.
+
+#### Option 1 Clone PX4, install Toolchain
 ```sh
-sudo usermod -a -G uucp $USER
+git clone --recurse-submodules -j8 https://github.com/PX4/Firmware.git
+source Firmware/Tools/setup/arch.sh # optionally append --gazebo
 ```
 
-Then log out and log in for changes to take effect.
+#### Option 2 only install Toolchain
+```sh
+wget https://raw.githubusercontent.com/PX4/Firmware/master/Tools/setup/arch.sh
+source arch.sh # optionally append --gazebo
+```
+
+## Manual Installation
```

Comment:
I think this section is not needed.

Actually it can be confusing. I read this document quickly and wondered why these steps were necessary after running the install script.

I expect Arch Linux users to be able to read the install script and run the commands manually if they prefer to. This is also less text to maintain. I suggest to move any useful comments into the install script and to remove this section.

Reply:
I agree. I just didn't want to wrecklessly remove all your instructions without having checked if they contain anything additional e.g. I don't remember if I would need `genromfs` for a specific case or it's anyways already contained somewhere else. I can go though.

Reply:
Agree too. I  would like the script to be sufficient and remove this altogether. If we need to add more docs to the script to explain them then go for it.

Reply:
I checked, it's really not needed. Even genromfs is not mandatory (anymore). I removed the section one by one while checking.